docker/kmq/Dockerfile

```diff
@@ -22,6 +22,10 @@ COPY sql_dumps/kmq-test-cached.sql sql_dumps/kmq-test-cached.sql
 COPY sql/ sql/
 COPY templates/ templates/
 
+COPY bin/ bin/
```

Comment:
Why do we need to copy it? Cant 'we just create an empty folder and download the latest release